src/app/(auth)/sign-in/actions.ts

```
@@ -33,6 +33,7 @@ export async function signInAction(
       },
     });
 
+     const redirectTo = inviteToken
     const redirectTo = inviteToken
```

Biome (2.3.13)

[error] Shouldn't redeclare 'redirectTo'. Consider to delete it or rename it.

'redirectTo' is defined here:


(lint/suspicious/noRedeclare)